src/work-space/workspace.resource.tsx

```diff
@@ -0,0 +1,20 @@
+import { Subject, Observable } from "rxjs";
+
+const workspaceItem = new Subject<WorkspaceTab>();
+
+export function newWorkspaceItem(item: WorkspaceTab) {
+  workspaceItem.next(item);
+}
+
+export function getNewWorkspaceItem(): Observable<WorkspaceTab> {
+  return workspaceItem.asObservable();
+}
+
+export type WorkspaceTab = {
```

Comment:
We need a way of naming types. I thought for a moment that you were exporting a React component. 

For interfaces, people commonly seem to prepend with I, e.g. IWorkSpace. It may be easiest to just append Type to end as that's what most of us have been doing. So, for now, rename this WorkspaceTabType

Reply:
I renamed it to WorkSpaceItem, as that's what it is.